src/commands/config/mod.rs

```diff
@@ -17,10 +17,40 @@ pub fn set_file_mode(file: &PathBuf) {
         .expect("could not set permissions on file");
 }
 
-pub fn global_config(email: String, api_key: String) -> Result<(), failure::Error> {
-    let s = GlobalUser { email, api_key };
+pub fn global_config(api_key: bool) -> Result<(), failure::Error> {
+    let mut user = GlobalUser {
```

Comment:
i assume this is where you were alluding to an enum refactor? i agree with that; there should be two constructors for this type. is there an issue?

Reply:
yup! i can file an issue for the refactor.